crates/autopilot/src/infra/persistence/dto/order.rs

```diff
@@ -22,6 +22,7 @@ pub struct Order {
     #[serde_as(as = "HexOrDecimalU256")]
     pub buy_amount: U256,
     pub protocol_fees: Vec<FeePolicy>,
+    pub created: u32,
```

Comment:
I know you didn't write this, but having a `dto` within persistence which is also used in an `reqwest` petition is really confusing.

What's more, are you sure we do not store this struct into a database in any way right? because those are new fields, and even if we store this as a plain `json`, when deserializing the old data it will crash because this is not `Option<T>` type.

Reply:
Oh I just saw the "important" in the PR description :facepalm: